contributing/code/maintenance.rst

```diff
@@ -0,0 +1,58 @@
+Maintenance
+===========
+
+During the lifetime of a minor version, new releases (patch versions) are
+published on a monthly basis. This document describes the boundaries of
+acceptable changes.
+
+**Bug fixes** are accepted under the following conditions:
```

Comment:
@alexislefebvre bug fixes can be merged in patch versions. This means that now 3.0.0 was released, the 3.0 branch can only have bug/security fixes. New features must only be merged in the unstable branch (master/3.1 at the moment).